gordo_components/builder/build_model.py

```diff
@@ -593,3 +586,31 @@ def provide_saved_model(
         logger.info(f"Writing model-location to model registry")
         disk_registry.write_key(model_register_dir, cache_key, model_location)
     return model_location
+
+
+def metrics_from_list(metric_list: Optional[List[str]] = None) -> List[Callable]:
```

Comment:
I know this is accepted already, so sorry for my nitpicking, but does not make sense to *add* the posibility to specify the whole function, like `sklearn.metrics.r2_score` ? The argument for allowing this is that pretty much everywhere else we are refering to functions we are refering to them with their full path, so it is not unlikely that a user would use the whole path (I certainly would). 
I think it is OK to also do what is done here, allowing the use of all metrics in `sklearn.metrics` without the full path. It can be a bit confusing after some time knowing where that is allowed and where it is not, but I think the convenience outweights that problem. But if we could say that "full paths are always allowed" that makes it a bit nicer at least(?)

Reply:
Good points.

I'm not sure though what other functions would be supported, ever. As we wrap specific scoring functions with expected signatures. I will happily update this function to do as you say otherwise. It just appears that the need for referring to other scoring functions would become quite rare indeed; as their signatures would need to be the same as what our wrapper expects, no?

Reply:
We also don't specify the full path to the dataset type. We would update that as well?

Reply:
> We also don't specify the full path to the dataset type. We would update that as well?

True, I meant that we specify full paths when we use types not defined by us. I don't think we should update the dataset type to full path, we reserve the right to move it wherever we want without that leaking out in the config file.

I don't know if we will ever refer to functions outside sklearn.metrics, but that is not actually the primary reason I want it. It is mostly a "consistency / aesthetic" thing, that everywhere else I refer to sklearn stuff I write `sklearn.preprocessing...`, and it is a bit weird that this particular place that is not possible